app/views/users/_dashboard_deck.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="block">
   <div class="block-image">
-    <%= link_to "/learn/#{@user.id}/#{deck.id}" do %>
+    <%= link_to "/users/#{@user.id}/#{deck.id}" do %>
       <%=image_tag image_path("temp/dashboard-img-#{rand(1..4)}.jpg") %>
     <% end %>
   </div>
   <div class="block-details">
-    <%= link_to "/learn/#{@user.id}/#{deck.id}", class: 'title-link' do %>
+    <%= link_to "/users/#{@user.id}/#{deck.id}", class: 'title-link' do %>
```

Comment:
ouuu thats right, its the resource that makes the helper not the routes itself. i confirmed my method doesnt have a helper either:

```
                         POST   /learn/:id/:deck_id(.:format)             learn#create_card_response
                   users GET    /users(.:format)                          users#index
```